Makefile

```
@@ -80,6 +82,60 @@ build: deps
 setup: deps hooks
 	@echo "Development environment setup complete"
 
+# Database migration targets using golang-migrate
+migrate:
```

checkmake (0.2.2)

[warning] Target body for "migrate" exceeds allowed length of 5 (8).

(maxbodylength)